### DISCUSSION TITLE
Stand Down Days - not crediting

### DISCUSSION THREAD

**Discussion by memphisplanthire** [stayonhire/feedback#25] [Feedback / Issues]
Hi team,

Not sure if it's just me, but I've just tried to apply a stand down day to an invoice and it's adding the amount on, not crediting it.

Thanks
![Stand down](https://github.com/stayonhire/feedback/assets/142187468/e7333240-7d75-4dbb-9ebb-82b4026a4e0a)

**Comment by twrayden** [comment]
Will be looking at this today, and will update here when fixed :+1:

**Comment by dylanhru** [comment]
Hi Memphis Team,

I can confirm stand-downs do now insert a crediting amount (i.e. negative sum):
![Screenshot 2023-08-15 at 8 54 44 am](https://github.com/stayonhire/feedback/assets/34927957/fc682c1f-4251-4e04-a856-6178d224a1c1)

If you have inserted an existing stand down into an invoice, this can be fixed by removing the invoice row and re-insterting it :)

**Other improvements made yesterday include:**
- Additional warning prompts added to 'deleting stand downs'
- Bug fix to stand down visibility (in billable items table) when an approved invoice is voided containing a stand down
- Bug fix editing stand-down invoice line item description, and line height visibility
- Update contacts sort to A-Z

**Improvements coming today/tomorrow:**
1. Accounts and tax rates will prefill on stand-downs
2. 'Refresh draft' button coming for invoices that have new on-hire hours (since drafting)
3. Bug fixes to template exporting dates 
4. Private notes column will be added to the billable items table
5. 'Applied' stand-downs will contain a link to the invoice they're applied to
